lib/util/metrics.js

```diff
@@ -0,0 +1,72 @@
+let metric = {
+	newConnectionsCount: 0,
+	reconnectionCount: 0,
+	closedConnectionCount: 0,
+	messagesCount: 0,
+	activeConnectionCount: 0,
+	errorConnectionCount: 0,
+	incomingQueueCount: 0,
+	outcomingQueueCount: 0,
```

Comment:
where are we updating these queueCounts?

Reply:
Aah not required now